backend/src/__tests__/canvasAPITests/rubricsAPI/updateRubricTests.test.ts

```diff
@@ -11,8 +11,9 @@ describe("updateRubric", () => {
   it("should make a PUT request to update an existing rubric in a specific course", async () => {
     // Arrange
     const request: UpdateRubricRequest = {
-      id: 123,
+      rubric_id: 123,
```

Comment:
Canvas has this field as id and not rubric_id

See: https://canvas.instructure.com/doc/api/rubrics.html#method.rubrics.update